nvdaHelper/cppjieba/cppjieba.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+This file is a part of the NVDA project.
```

Comment:
please copy our copyright headers directly from
https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/copyrightHeaders.md
and use that format